tern/analyze/default/container/run.py

```diff
@@ -73,7 +74,12 @@ def execute_image(args):
     image_string = extract_image(args)
     # If the image has been extracted, load the metadata
     if image_string:
-        full_image = cimage.load_full_image(image_string)
+        full_image = cimage.load_full_image(image_string, args.load_until_layer)
+        if full_image.load_until_layer > full_image.total_layers:
```

Comment:
I'm unsure about this part, I think this should be raised as exception by DockerImage.load_full_image (which I think should be now renamed to load_image() to avoid confusion in the future. What do you think @rnjudge?